.github/workflows/linters.yml

```diff
@@ -0,0 +1,49 @@
+name: Linters
+
+on:
+  pull_request:
+    branches: 
+     - "**"
+     
+env:
+  TARGET_BRANCH: ${{ github.base_ref }}
+
+jobs:
+  run-linters:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Run npm install
+      run: npm install --save-dev
+
+    - name: Fetch target branch
+      run: git fetch -n origin ${TARGET_BRANCH}
+            
+    - name: Run phpcs linter 
+      run: |
+        FILES_DIFF=$(git diff origin/${TARGET_BRANCH} --no-commit-id --name-only -r -- '*.php')
+        if [ -n "$FILES_DIFF" ]; then
+          ./bin/phpcs.phar $FILES_DIFF --standard=phpcs-ruleset.xml
+        fi
+
+    - name: Run stylelint linter
+      #This step will always run regardless of previous step's status
+      #without if: ${{ always() }}, if the previous step is failure,
+      #the remaining step will not be executed. 
+      if: ${{ always() }} 
+      run: |
+        FILES_DIFF=$(git diff origin/${TARGET_BRANCH} --no-commit-id --name-only -r -- '*.scss')
+        if [ -n "$FILES_DIFF" ]; then
+          npx stylelint --config .stylelintrc $(git diff origin/${TARGET_BRANCH} --no-commit-id --name-only -r -- '*.scss') 
```

Comment:
@erawat , Any reason why the FILES_DIFF variable is not used here?
i.e 
```php
npx stylelint --config .stylelintrc FILES_DIFF
```

Reply:
@tunbola  there is no reason, I will make a change and update the PR.